controllers/survey.py

```
@@ -498,8 +488,6 @@ def series_export_word(widget_list, lang_dict, title, logo):
         try:
             AddRow(*line)
         except:
-            if DEBUG:
```

This will swallow errors - so either you keep the raise if settings.base.debug, or at least log the exception.

The except does not specify an exception type - would this be a document problem or a problem with the "line"? If it's a document problem (which is likely), then all lines would fail, so "pass" is not a valid recovery. If it is a line problem, then adding a placeholder line specifying the error would be the better option.